docker-compose.yml

```diff
@@ -18,11 +44,22 @@ services:
       HTTPS_CERT_FULLCHAIN: /etc/ssl/fullchain.pem
       HTTPS_CERT_PRIVKEY: /etc/ssl/privkey.pem
       INTERACTIVE: "false"
+      MEDIASOUP_MIN_PORT: 40000
+      MEDIASOUP_MAX_PORT: 40050
+      MEDIASOUP_ANNOUNCED_IP: 10.20.30.12
     ports:
       - "4443:4443"
+      # This should be the same as MEDIASOUP_MIN_PORT and MEDIASOUP_MAX_PORT
+      # You might neeed to increase this number depending on how many client you are connecting.
+      # (Keep the ports range small to improve container statup time)
+      - "40000-40050:40000-40050"
+      - "40000-40050:40000-40050/udp"
     volumes:
       - dialog:/code
     working_dir: /code
+    networks:
+      hubs_network:
+        ipv4_address: 10.20.30.12
```

Comment:
Why is it necessary to fix the address to a static IP?

Reply:
You need to listen in a specific IP address, you can't listen in all interfaces. Is there a better way to assign the current container IP without making it static?

Reply:
I don’t follow. Does it need to be the same between orchestrations? Who is listening?

Reply:
If we just need to know the current container IP, then **#** `hostname -i` should do the trick.